doc/source/whatsnew/v2.0.0.rst

```diff
@@ -1075,8 +1075,8 @@ Metadata
 
 Other
 ^^^^^
+- Add future warnings in :meth:`DataFrameGroupBy.fillna` and :meth:`SeriesGroupBy.fillna` allowing only keyword arguments to be passed except ``value`` (:issue:`48395`)
```

Comment:
move to section starting on L564 and reword like the others, so maybe "Deprecated...." instead of mentioning FutureWarning.